mkdocs.yml

```
@@ -165,7 +165,9 @@ nav:
     - Motion and animation:               'architecture/motion-animation.md'
     - Theming:                            'architecture/theming.md'
     - UI state:                           'architecture/ui-state.md'
-  - Contributing:                         'CONTRIBUTING.md'
+  - Contributing:
+    - Contributing: CONTRIBUTING.md
```

So we don't have `contributing` twice.

```suggestion
    - Guidelines: CONTRIBUTING.md
```

---

Done.